lib/registries/addon/branded/new/controller.ts

```diff
@@ -0,0 +1,88 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { timeout } from 'ember-concurrency';
+import { task } from 'ember-concurrency-decorators';
+import DS from 'ember-data';
+
+import NodeModel from 'ember-osf-web/models/node';
+import RegistrationSchemaModel from 'ember-osf-web/models/registration-schema';
+import Analytics from 'ember-osf-web/services/analytics';
+import CurrentUserService from 'ember-osf-web/services/current-user';
+import captureException, { getApiErrorMessage } from 'ember-osf-web/utils/capture-exception';
+import Toast from 'ember-toastr/services/toast';
+
+export default class BrandedRegistriesNewSubmissionController extends Controller {
+    @service analytics!: Analytics;
+    @service currentUser!: CurrentUserService;
+    @service store!: DS.Store;
+    @service toast!: Toast;
+
+    @tracked selectedProject?: NodeModel = undefined;
+    @tracked selectedSchema?: RegistrationSchemaModel = undefined;
+    @tracked schemaOptions?: RegistrationSchemaModel[] = [];
+    @tracked projectOptions?: NodeModel[] = [];
```

Comment:
✨

Reply:
oh wait nitpicks:
- if you declare with `?`, don't need to initialize to `undefined`
- if you initialize to a defined value, probably don't need `?`